Goobi/newpages/ProjekteBearbeiten.jsp

```diff
@@ -223,7 +223,7 @@
 																	monthYearRowClass="monthYearRowClass"
 																	imageLocation="/newpages/images/calendarImages"
 																	popupButtonImageUrl="/newpages/images/calendarImages/calendar.gif" />
-																<%--popupButtonImageUrl="images/calender.gif"--%>
+																<%--popupButtonImageUrl="images/calendar.gif"--%>
```

Comment:
A filename must not be correct in English language as long this file exists. If you want to translate the file name, you must rename this file too.